test_conformance/api/test_platform.cpp

```diff
@@ -230,14 +230,22 @@ int test_get_platform_ids(cl_device_id deviceID, cl_context context, cl_command_
     }
 
     err = clGetDeviceIDs(platforms[p], CL_DEVICE_TYPE_ALL, 0, NULL, &num_devices);
-    test_error(err, "clGetDeviceIDs size failed.\n");
+    test_error(err, "clGetDeviceIDs failed.\n");
+    if(num_devices == 0)
+    {
+        log_error("clGetDeviceIDs must return at least one device\n");
+        total_errors++;
+    }
+    test_assert_error(num_devices > 1,
+                      "clGetDeviceIDs must return at least one device");
```

Comment:
Condition should either 'num_devices >= 1' or 'num_devices > 0' and not 'num_devices > 1'.

Reply:
@lakshmih this will be changed to ">=1"

Reply:
Do we need both the `log_error` and the `test_assert_error`?  Seems redundant.  Can we remove the `test_assert_error` case?